app/controllers/products_controller.rb

```diff
@@ -25,6 +25,37 @@ def create
   end
 
   def edit
+    @product = Product.find_by(id: params[:id])
+    driver_id = params[:id]
```

Comment:
driver_id?

Reply:
Also, can user Jessica's find_product method. So should be able to remove the @product = Product.find_by....